package.json

```diff
@@ -10,8 +14,10 @@
     "validate-setup": "node scripts/validateSetup.js",
     "test-integration": "sh scripts/tests/testIntegration.sh",
     "test-unit": "sh scripts/tests/testUnit.sh",
-    "test": "npm run lint && npm run test-integration && npm run test-unit",
-    "generate-api-schema": "node utils/generateAPISchema.js"
+    "test": "npm run lint && npm run test-integration && npm run test-unit && bundle exec rake cucumber && bundle exec rake rspec && bundle exec rake jasmine:ci",
```

Comment:
Is cucumber.js needs to be installed for the execution of this script?

Reply:
How does rspec and jasmine:ci helps here?

Reply:
I am not too much familiar with the code written. I saw some of the code working same as I wanted that's why I just copy pasted from that site.
I will surely read about all the terms I have used there.

Reply:
Also, I followed the steps as listed there, and there was no instruction of installing cucumber.js. So, I guess it is not required to install cucumber.js

Reply:
I think node version checking should be done while validating the setup

I think ` bundle exec rake cucumber && bundle exec rake rspec && bundle exec rake jasmine:ci",` is not required, we are not using jasmine or cucumber or rspc?

Reply:
https://medium.com/@faith__ngetich/locking-down-a-project-to-a-specific-node-version-using-nvmrc-and-or-engines-e5fd19144245

@prakashchoudhary07  I referred to this site for completing this task and yes, I removed that line you have suggested as it was an unnecessary test.